setup.py

```diff
@@ -41,8 +42,10 @@
             "black==20.8b1",
             "flake8==3.9.0",
             "flake8-import-order==0.18.1",
+            "mypy==0.982",
             "pytest==6.2.2",
```

Comment:
I just remembered: this version of `pytest` is incompatible with Python 3.10. The bug is fixed on version 6.2.5 but perhaps you should just upgrade to the latest version while your at it.

Reply:
thanks
I'll add a test for 3.11 just to see if we have any upper bound on the python version too...
I know we  people internally already running 3.11...